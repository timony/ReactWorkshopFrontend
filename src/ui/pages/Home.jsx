import React from 'react';
import {homeLabel} from '../../services/functions';

export class Home extends React.Component {
  render() {
    const element = (
      <div>{homeLabel()}</div>
    );

    return (
      element
    );
  }
}

export default Home;
