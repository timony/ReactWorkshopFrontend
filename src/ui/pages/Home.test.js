import React from 'react';
import renderer from 'react-test-renderer';
import {Home} from './Home';

describe('Home dump component', () => {
  it('should render with defaults', () => {
    const component = renderer.create(<Home/>);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
