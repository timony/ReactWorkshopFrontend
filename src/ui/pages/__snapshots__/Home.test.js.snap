// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home dump component should render with defaults 1`] = `
<div>
  Home page from function
</div>
`;
